moto/rds3/models/db_snapshot.py

```diff
@@ -147,7 +147,7 @@ def delete_db_snapshot(self, db_snapshot_identifier):
         snapshot.delete_events()
         return self.db_snapshots.pop(db_snapshot_identifier)
 
-    def describe_db_snapshots(self, db_instance_identifier=None, db_snapshot_identifier=None, snapshot_type=None):
+    def describe_db_snapshots(self, db_instance_identifier=None, db_snapshot_identifier=None, snapshot_type=None, marker=None, max_records=None):
```

Comment:
Handle input validation for max_records
Also, should this closely mimic aws cli which allows between 20-100?
```
MaxRecords. Must be between 20 and 100
```